css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+body {
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+}
+
+* {
+  text-align: center;
+}
+
+.container {
+  padding-top: 3rem;
+  color: white;
+  max-width: 75%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  letter-spacing: 0.1rem;
+  justify-content: center;
```

Comment:
You may not always want your row to be centering its content like this. I would move this to another class, named `justify-center`